index.js

```diff
@@ -138,47 +188,64 @@ function nowServing(line) {
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
+let tickCounter = 1
 
+function takeATicketNumber(line) {
+  line.push(tickCounter)
+  tickCounter ++
+  return `Welcome. You are ticket number ${line[line.length-1]}.`
```

Comment:
This can be simplified to:
```suggestion
  return `Welcome. You are ticket number ${tickCounter - 1}.`
```